### PR TITLE
AGPUSH-1260 : add new apns keys

### PR DESCRIPTION
**Pull request by sebastienblanc**
@cvasilak @matzew @corinnekrych can you review/merge ?

### CODE REVIEW

File: src/test/java/org/jboss/aerogear/unifiedpush/message/UnifiedMessageTest.java [hunk 1]
```diff
@@ -128,7 +130,9 @@ public void noContentAvailable() {
     @Test
     public void actionCategoryTest() {
         UnifiedMessage unifiedMessage = UnifiedMessage.withMessage()
-                .actionCategory("myInteractiveNotification")
```
Comment (matzew) [inline src/test/java/org/jboss/aerogear/unifiedpush/message/UnifiedMessageTest.java]: can we have a test, that combines both, Windows and Apns builders?

Comment (sebastienblanc) [inline src/test/java/org/jboss/aerogear/unifiedpush/message/UnifiedMessageTest.java]: test added

File: src/test/java/org/jboss/aerogear/unifiedpush/message/UnifiedMessageTest.java [hunk 2]
```diff
@@ -204,5 +208,19 @@ public void windowsRawMessage() {
...
+        UnifiedMessage unifiedMessage = UnifiedMessage.withMessage()
+                .windows()
+                  .raw()
+                  .build()
+                .apns()
+                  .contentAvailable()
+                  .build()
+                .build();
+        assertEquals(Type.raw, unifiedMessage.getMessage().getObject().getWindows().getType());
+        assertEquals(true, unifiedMessage.getMessage().getObject().getApns().isContentAvailable());
```
Comment (matzew) [inline src/test/java/org/jboss/aerogear/unifiedpush/message/UnifiedMessageTest.java]: let's, at some point, get rid of that and use AssertJ instead  - but that should be a different PR

**Comment by matzew** [comment]
:+1:

**Comment by cvasilak** [comment]
@sebastienblanc opened https://github.com/sebastienblanc/aerogear-unified-push-java-client/pull/1 for fixing javadoc warnings during generation

**Comment by cvasilak** [comment]
according to [Apple push localized guide](https://developer.apple.com/library/ios/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/Chapters/ApplePushService.html#//apple_ref/doc/uid/TP40008194-CH100-SW21) and the [APNS source](https://github.com/notnoop/java-apns/blob/master/src/main/java/com/notnoop/apns/PayloadBuilder.java) seems we are on the right track.

nice work @sebastienblanc and @matzew 

@corinnekrych may I merge ?

**Comment by lholmquist** [comment]
@sebastienblanc is there anything on the Node.js sender side that needs to be done here,  just a doc update. didn't see a JIRA for it

**Comment by sebastienblanc** [comment]
@lholmquist Ah right, I saw https://issues.jboss.org/browse/AGPUSH-1261 but since we change the structure, let me open (or reopen this one) one ?

**Comment by lholmquist** [comment]
@sebastienblanc maybe reopen that one.  Do we also need one for the Windows format?

**Comment by sebastienblanc** [comment]
@lholmquist yes we also need it for windows, that why I created a new, more general, ticket : https://issues.jboss.org/browse/AGPUSH-1318

**Comment by lholmquist** [comment]
@sebastienblanc cool beans
